learners/reference.md

```diff
@@ -1,15 +1,105 @@
 ---
-title: 'FIXME'
+title: Reference
 ---
 
 ## Glossary
 
-## Reference
+*The definitions below are modified from the [Carpentries
+Glosario](https://glosario.carpentries.org/) (CC-BY-4.0)*
+
+[aggregation](https://glosario.carpentries.org/en/#aggregation)
+:   To combine many values into one, e.g., by summing a set of numbers.
+
+[field](https://glosario.carpentries.org/en/#field)
+:   A component of a [record](#record) containing a single value. Every
+    record in a database [table](#table) has the same fields.
+
+[filter](https://glosario.carpentries.org/en/#filter)
+:   To choose a set of [records](#record) (i.e., rows of a table) based
+    on the values they contain.
+
+[full join](https://glosario.carpentries.org/en/#full_join)
+:   A [join](#join) that returns all rows and all columns from two
+    tables A and B. Where the [keys](#key) of A and B match, values are
+    combined; where they do not, missing values from either table are
+    filled with [null](#null), NA, or some other [missing
+    value](#missing_value) signifier.
+
+[group](https://glosario.carpentries.org/en/#group)
+:   To divide data into subsets according to a set of criteria while
+    leaving records in a single structure.
+
+[inner join](https://glosario.carpentries.org/en/#inner_join)
+:   A [join](#join) that returns the combination of rows from two
+    tables, A and B, whose [keys](#key) exist in both tables.
+
+[join](https://glosario.carpentries.org/en/#join)
+:   One of several operations that combine values from two
+    [tables](#table).
+
+[key](https://glosario.carpentries.org/en/#key)
+:   A [field](#field) or combination of fields whose value(s)
+    uniquely identify a [record](#record) within a [table](#table) or
+    dataset. Keys are often used to select specific records and in
+    [joins](#join).
+
+[left join](https://glosario.carpentries.org/en/#left_join)
+:   A [join](#join) that combines data from two tables, A and B, where
+    [keys](#key) in table A match keys in table B, [fields](#field) are
+    concatenated. Where a key in table A does \*not\* match a key in
+    table B, columns from table B are filled with [null](#null), NA, or
+    some other [missing value](#missing_value). Keys from table B that
+    do not match keys from table A are excluded for the result.
+
+[missing value](https://glosario.carpentries.org/en/#missing_value)
+:   A special value such as [null](#null) or NA used to indicate the
+    absence of data. Missing values can signal that data was not
+    collected or that the data did not exist in the first place (e.g.,
+    the middle name of someone who does not have one).
+
+[null](https://glosario.carpentries.org/en/#null)
+:   A special value used to represent a missing object.
+
+[record](https://glosario.carpentries.org/en/#record)
```

Comment:
```suggestion
[[__record__]{#record}](https://glosario.carpentries.org/en/#record)
```

Reply:
Fixed the links, thanks! This is what I get for not actually building the lesson.